CHANGELOG.md

```diff
@@ -7,3 +7,4 @@ All notable changes to this project will be documented in this file.
 *   First release with several profiles moved from psick module
 *   General Puppet Lint cleanups
 *   CI setup
+*   Unit tests for each class and define
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/example42/puppet-psick_profile/pullRequest?prid=9480555)
```suggestion
* Unit tests for each class and define
```